compiler/src/main/scala/org/scalaexercises/evaluator/StringCompiler.scala

```diff
@@ -0,0 +1,133 @@
+/*
+ * scala-exercises-evaluator-compiler
+ * Copyright (C) 2015-2016 47 Degrees, LLC. <http://www.47deg.com>
+ */
+
+package org.scalaexercises.evaluator
+
+import java.io.File
+
+import scala.tools.nsc.{Global, Settings}
```

Comment:
Same here about twitter-eval or here only depends on which one has some of the modified twitter-eval code.